<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="5" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-28T11:41:41 IST" name="Default suite" finished-at="2022-11-28T11:42:19 IST" duration-ms="37628">
    <groups>
    </groups>
    <test started-at="2022-11-28T11:41:41 IST" name="Default test" finished-at="2022-11-28T11:42:19 IST" duration-ms="37628">
      <class name="TestCases.AmazonAppTestCases">
        <test-method signature="setBrowserrr()[pri:0, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:41:41 IST" name="setBrowserrr" finished-at="2022-11-28T11:41:54 IST" duration-ms="12260" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The expected url is not similar as Actual 
Amazon.in
ActualTitle: Amazon Sign In expected [Amazon Sign In] but found [Amazon.in]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The expected url is not similar as Actual 
Amazon.in
ActualTitle: Amazon Sign In expected [Amazon Sign In] but found [Amazon.in]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at TestCases.AmazonAppTestCases.setBrowserrr(AmazonAppTestCases.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setBrowserrr -->
        <test-method signature="verifyLoginPage()[pri:2, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:41:54 IST" name="verifyLoginPage" finished-at="2022-11-28T11:41:55 IST" duration-ms="1044" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginPage -->
        <test-method signature="verifyHomePage()[pri:3, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:41:55 IST" name="verifyHomePage" finished-at="2022-11-28T11:42:00 IST" duration-ms="5362" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHomePage -->
        <test-method signature="selectitem()[pri:4, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:42:00 IST" name="selectitem" finished-at="2022-11-28T11:42:00 IST" duration-ms="157" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- selectitem -->
        <test-method signature="addcart()[pri:5, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:42:00 IST" name="addcart" finished-at="2022-11-28T11:42:14 IST" duration-ms="13138" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addcart -->
        <test-method signature="proceedtoBuy()[pri:6, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:42:14 IST" name="proceedtoBuy" finished-at="2022-11-28T11:42:19 IST" duration-ms="5434" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- proceedtoBuy -->
        <test-method signature="skip()[pri:7, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:42:19 IST" name="skip" finished-at="2022-11-28T11:42:19 IST" duration-ms="7" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[skip this exception]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: skip this exception
at TestCases.AmazonAppTestCases.skip(AmazonAppTestCases.java:124)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- skip -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:TestCases.AmazonAppTestCases@305ffe9e]" started-at="2022-11-28T11:42:19 IST" name="teardown" finished-at="2022-11-28T11:42:19 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- TestCases.AmazonAppTestCases -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
